### ISSUE TITLE
A11y_Model Builder_Add Data_advanced data option_Usability: "Advanced data options" control is defined as link but it is button.

### ISSUE THREAD

**Issue by Harika1702** [dotnet/machinelearning-modelbuilder#2657]
### GitHub Tags
#A11ySev4; #A11yTCS; #A11yUsable: #DesktopApp; #Win11; #Visual Studio Client; #SH_ModelBuilder_Win32_May2023; #FTP; #Win32; #Benchmark;

### Environment
Application Name: Model Builder
Microsoft Visual Studio Enterprise 2022 (64-bit) - Current
Version 17.6.1
Microsoft Windows 11 Enterprise Build 22624.1755

### Repro Steps
1. Launch the Visual Studio with at least one of the .NET workloads.
2. Tab till enable ML.NET Model Builder (Tools -> Options -> Environment -> Preview Features) and select it.
3. Create a new .NET project or open an existing .NET project.
4. Open context menu (Right-click) of the project in the solution explorer.
5. Tab till Add and then tab till Machine Learning Model and press enter.
6. Model Builder dialog will open. Complete Scenario and Environment setup steps.
7. Add Data screen will appear. Tab till Advance data option.
8. Now verify that "Advanced data options" control is defined as link but it is button.

### Actual Result
"Advanced data options" control is defined as link but it is button.
Consume tab -- Copy and "Add to solution" Controls are defined as link but those are buttons.
**Note: We are facing similar issues throughout the application.**

### Expected Result
"Advanced data options" control should be define as "Button".

### User Impact
Screen reader users will get confused for if the role for ""Advanced data options".
### Attachment
<img width="960" alt="Bug_Role" src="https://github.com/dotnet/machinelearning-modelbuilder/assets/100270074/0cc96167-b83c-47db-95c9-87550dcd480a">
**<img width="960" alt="Bug_Link" src="https://github.com/dotnet/machinelearning-modelbuilder/assets/100270074/5eae0c50-99f7-405c-9c11-588e1864c4de">**
<img width="960" alt="Bug _Button" src="https://github.com/dotnet/machinelearning-modelbuilder/assets/100270074/6cb7926b-03e1-4d60-b19b-b8727ba999e5">

**Comment by SachinChavhan317** [comment]
Verified the bug on below model builder version.
Version:17.17.0.2332602
The issue is fixed, hence closing the bug. #Closed;
Adding fixed attachment.

![2657_Fixed1](https://github.com/dotnet/machinelearning-modelbuilder/assets/86241494/e23f0917-fee2-47f7-89cf-e92dce59442a)
![2657_Fixed2](https://github.com/dotnet/machinelearning-modelbuilder/assets/86241494/2126c416-c395-4a1d-b262-86e303d5dcb4)
![2657_Fixed3](https://github.com/dotnet/machinelearning-modelbuilder/assets/86241494/9d4b03b9-1392-4f7f-b8f1-2e249504955b)
![2657_Fixed4](https://github.com/dotnet/machinelearning-modelbuilder/assets/86241494/90c951d2-c37c-47f8-a829-da5104e0b05a)
